code/datums/mutations.dm

```diff
@@ -147,6 +168,10 @@ var/thanks_tobba = 'icons/fonts/runescape_uf.ttf'
 	if(..())
 		return
 	owner.status_flags |= CANSTUN | CANWEAKEN | CANPARALYSE | CANPUSH
+	owner.client.prefs.features["mcolor"] = naturalcolor
```

Comment:
because it doesn't work right, and even though it changes their prefs, it won't carry over between rounds because they're not saving it. The only potential edge case is if an admin forcibly ghosts them and sends them back to the main screen while they were a hulk, and then they save the color.